### DISCUSSION TITLE
Graves opening

### DISCUSSION THREAD

**Discussion by daretmavi** [GriefPrevention/GriefPrevention#2056] [Q&A]
Is it possible to open graves from https://vanillatweaks.net/picker/datapacks/ (Datapack "Graves").

If player dies in claimed area, he can not open his grave, because he doesn't have trust permission and the owner can't open it as well, because grave robbing is disabled.
Only OP can get grave key.

**Comment by daretmavi** [comment]
Or ... does someone know better graves plugin, that works good with GriefPrevention?

**Comment by Jikoo** [comment]
An addon could theoretically support this, but you should use a plugin anyway. Datapacks generally are less efficient than plugins because they repeatedly have to compare and check values every tick rather than being triggered by an event fired by the server at the correct time. Just google "grave plugin bukkit" and you'll see that there are several dozen options. I imagine one has support for GP.
